hack/test-cmd.sh

```diff
@@ -352,12 +345,19 @@ for test in "${tests[@]}"; do
   # switch back to a standard identity. This prevents individual tests from changing contexts and messing up other tests
   oc project ${CLUSTER_ADMIN_CONTEXT}
   oc new-project "cmd-${name}"
-  ${test}
+  if ! ${test}; then
+    failed="true"
+    tail -40 "${LOG_DIR}/openshift.log"
+  fi
   oc project ${CLUSTER_ADMIN_CONTEXT}
   oc delete project "cmd-${name}"
   cp ${KUBECONFIG}{.bak,}  # since nothing ever gets deleted from kubeconfig, reset it
 done
 
+if [[ -n "${failed:-}" ]]; then
```

Comment:
Without my bash manual, I don't know what this does.  If this doesn't fail properly, we just lost all the tests.  **be sure**